### DISCUSSION TITLE
OffsetY and OffsetX

### DISCUSSION THREAD

**Discussion by colstewart** [gilbarbara/react-floater#93] [Ideas]
Hi there,

I see that we can pass an offset to the Floater, which is the amount of pixels it should be away from the target. Is there a way to pass an offsetY and offsetX? I only see the `offset` prop which is currently offsetY (from the top). Would be great to be able to control both y and x

**Comment by gilbarbara** [comment]
hey @colstewart 

I have enabled Discussions and moved your issue to **Ideas**.

The `offset` is relative to the mais axis, and it could be Y or X (for left/right floaters). And you can also specify the cross-axis position (start, end) with the `placement` prop.

While it's easy to control the whole floater distance from the target (with the `offset` prop), the arrow position is calculated automatically by [popper](https://popper.js.org/) which is used for the overall positioning for this library.

You can change the [modifiers](https://github.com/gilbarbara/react-floater/blob/main/src/index.tsx#L219) used by popper with the `options` prop for maximum control.
